tests/test_telemetry.py

```diff
@@ -371,6 +370,62 @@ def test_app_dependencies_loaded(self):
             if not seen:
                 raise Exception(dependency + " not recieved in app-dependencies-loaded message")
 
+    def setup_app_dependency_loaded_not_sent_dependency_collection_disabled(self):
+        weblog.get("/enable_product")
```

Comment:
what is this for?

Reply:
To enable products i.e. GET /enable_product

This endpoint should enable all products 

DD_PROFILING_ENABLED=true

DD_APPSEC_ENABLED=true 

DD_DYNAMIC_INSTRUMENTATION_ENABLED=true

Reply:
enable product vis application code which is feasible in Python